xbmc/platform/darwin/osx/SDL/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+if(SDL_FOUND)
```

Comment:
I would remove the `if` statement here. Anything in `xbmc/platform/darwin/osx/SDL` must be ignored if SDL isn't found. That means the `if(SOURCES)` can also be removed, right?

Reply:
The issue is our cmake treedata cant handle states like that, its just all pulled in and folders parsed for CMakeLists.txt files.

The other option would be to not add xbmc/platform/darwin/osx/SDL to the cmake/treedata/osx/subdirs.txt and do the inclusion behind the same guard in xbmc/platform/darwin/osx/CMakeLists.txt

Reply:
OK, I think it's fine here